nixos/modules/services/networking/lxd-image-server.nix

```diff
@@ -56,11 +56,9 @@ in
         frequency = "daily";
         rotate = 21;
         create = "755 lxd-image-server ${cfg.group}";
-        missingok = true;
```

Comment:
please put changes to other modules in a separate pr

Reply:
This is a default value, so this doesn't change anything -- it just reflects defaults that had always been there.
As there is compatibility I can drop all the revamp though and open a later PR if you prefer, but folks using it will get a wall of warnings until it's converted. What would you like?

Reply:
nevermind, i was thinking about some other PR while looking at the files